Testscripts/Windows/PERF-SCHEDULER.ps1

```diff
@@ -0,0 +1,208 @@
+        Write-LogInfo ("`n**************************************************************************`n"+$CurrentTestData.testName+" RESULTS...`n**************************************************************************")
+        Write-Host ($schbenchDataCsv | Format-Table * | Out-String)
+        #endregion Show results*.csv
+
+        ##################################################################################################################
+        #region Parse log then upload the test results to database
+        ##################################################################################################################
+        $LogContents = @()
+        $LogContents += Get-Content -Path "$LogDir\results_hackbench.log"
+        $LogContents += Get-Content -Path "$LogDir\results_schbench.log"
+
+        if ($testResult -eq "PASS") {
+            $TestCaseName = $GlobalConfig.Global.$TestPlatform.ResultsDatabase.testTag
+            if (!$TestCaseName) {
+                $TestCaseName = $CurrentTestData.testName
+            }
+            $HostOs =  $(Get-Content "$LogDir\VM_properties.csv" | Select-String "Host Version"| ForEach-Object{$_ -replace ",Host Version,",""})
+            $GuestDistro = $(Get-Content "$LogDir\VM_properties.csv" | Select-String "OS type"| ForEach-Object{$_ -replace ",OS type,",""})
+            $KernelVersion = $(Get-Content "$LogDir\VM_properties.csv" | Select-String "Kernel version"| ForEach-Object{$_ -replace ",Kernel version,",""})
+            $TestDate = $(Get-Date -Format yyyy-MM-dd)
+
+            Write-LogInfo "Generating the performance data for database insertion"
+            foreach ($LogCon in $LogContents) {
+                for ($i = 1; $i -lt $LogCon.Count; $i++) {
+                    $Line = $LogCon[$i].Trim() -split '\s+'
+                    $resultMap = @{}
+                    $resultMap["TestCaseName"] = $TestCaseName
+                    $resultMap["TestDate"] = $TestDate
+                    $resultMap["HostType"] = $TestPlatform
+                    $resultMap["HostBy"] = $CurrentTestData.SetupConfig.TestLocation
+                    $resultMap["HostOS"] = $HostOs
+                    $resultMap["GuestOS"] = $GuestDistro
+                    $resultMap["InstanceSize"] = $clientVMData.InstanceSize
+                    $resultMap["KernelVersion"] = $KernelVersion
+                    if ($line[0] -imatch "hackbench") {
+                        $resultMap["TestMode"] = $($Line[0])
+                        $resultMap["DataSize_bytes"] = [Decimal]$($Line[1])
+                        $resultMap["Loops"] = [Decimal]$($Line[2])
+                        $resultMap["Groups"] = [Decimal]$($Line[3])
+                        $resultMap["HackbenchType"] = $($Line[5])
+                        $resultMap["Latency_sec"] = [Decimal]$($Line[6])
+                    } elseif ($line[0] -imatch "schbench") {
+                        $resultMap["TestMode"] = $($Line[0])
```

Comment:
$resultMap["TestMode"] = $($Line[0]) is no difference for schbench and hackbench, move it to common place?